src/applications/mhv/medications/containers/Prescriptions.jsx

```diff
@@ -168,6 +172,17 @@ const Prescriptions = () => {
 
   useEffect(
     () => {
+      if (!isLoading) {
+        if (prescriptionsApiError && isFirstRender) {
+          history.replace('/about');
+          return;
+        }
+        if (isFirstRender === undefined) {
+          setIsFirstRender(true);
+        } else {
+          setIsFirstRender(false);
+        }
+      }
```

Comment:
just curious, what the use case here?

Reply:
@agravell047 I set the flag to redirect the page for the first API call only.